### PR TITLE
386 draggable mobile nav

### LINKED ISSUE #386: Main nav is 'draggable' on mobile devices
Most notable on Android, navigation can be dragged by around 60%, affects both Samsung Internet and Chrome.

iOS devices can be dragged by approx 4 pixels, hardly noticeable but still there.

img_5720

Issue observed in:

- [x] Mobile Safari
- [x] Samsung Internet

### PR DESCRIPTION
**Pull request by nvasileiadis**
To sum up, the Transforms affect the footer alignment, the overflows fix the unwanted scrolling and hide the extra white space, the change in the twig file hides the "aaa" that used to appear next to the footer.

Closes #386 #458

### CODE REVIEW

File: stylesheets/_component.footer.scss [hunk 1]
```diff
@@ -28,8 +28,8 @@
     }
 
     .open-nav & {
-        -webkit-transform: translate3d($nav-primary-width, 0, 0) scale3d(1, 1, 1);
-        transform: translate3d($nav-primary-width, 0, 0) scale3d(1, 1, 1);
+        //-webkit-transform: translate3d($nav-primary-width, 0, 0) scale3d(1, 1, 1);
```
Comment (jamesjacobs) [inline stylesheets/_component.footer.scss]: Can we remove these commented lines?

File: stylesheets/_component.footer.scss [hunk 2]
```diff
@@ -28,8 +28,8 @@
...
+        //transform: translate3d($nav-primary-width, 0, 0) scale3d(1, 1, 1);
```
Comment (jamesjacobs) [inline stylesheets/_component.footer.scss]: Remove if no longer needed

File: stylesheets/_component.footer.scss [hunk 3]
```diff
@@ -39,8 +39,8 @@
     }
 
     .open-help & {
-        -webkit-transform: translate3d(-300px, 0, 0) scale3d(1, 1, 1);
-        transform: translate3d(-300px, 0, 0) scale3d(1, 1, 1);
+        //-webkit-transform: translate3d(-300px, 0, 0) scale3d(1, 1, 1);
```
Comment (jamesjacobs) [inline stylesheets/_component.footer.scss]: Remove if no longer needed

File: stylesheets/_component.footer.scss [hunk 4]
```diff
@@ -39,8 +39,8 @@
...
+        //transform: translate3d(-300px, 0, 0) scale3d(1, 1, 1);
```
Comment (jamesjacobs) [inline stylesheets/_component.footer.scss]: Remove if no longer needed

File: views/pulsar/layouts/base.html.twig [hunk 5]
```diff
@@ -166,6 +166,6 @@
 {% block modals %}{% endblock %}
 
 {% block javascript %}{% endblock %}
-{% block tab_javascript %}aaa{% endblock %}
+{# {% block tab_javascript %}aaa{% endblock %} #}
```
Comment (jamesjacobs) [inline views/pulsar/layouts/base.html.twig]: Remove?

**Comment by nvasileiadis** [comment]
Hey @jamesjacobs can you please have a second look? I believe the issues has been fixed.

**Pull request merged**